README.md

```diff
@@ -2,6 +2,24 @@
 
 The Tax Foundation’s publication [Capital Cost Recovery across the OECD](https://taxfoundation.org/publications/capital-cost-recovery-across-the-oecd/) shows how capital allowances compare across OECD countries and how they have developed since 2000. The dataset we compiled for the years 1979 to 2020 is made available as a resource for research.
 
+## Latest Release
+The current version of this project is 1.1.0 (2021-03-31).
+
+## Installation Directions
+You can analyze the data on your personal computer by using R.
+
+### Cloning the GitHub repository
+To install capital-cost-recovery, run the following in the folder you wish to store the files:
+
+git clone https://github.com/TaxFoundation/capital-cost-recovery
+cd capital-cost-recovery
```

Comment:
```recommendation
git clone https://github.com/TaxFoundation/capital-cost-recovery
cd capital-cost-recovery
```

Reply:
Not sure I understand what exactly you mean here.

Reply:
It's a recommendation to put the text in code blocks (three backticks above and below) so that it looks like this...

<img width="481" alt="image" src="https://user-images.githubusercontent.com/2487968/125120490-7a972300-e0c0-11eb-90d8-bef6a0f8191d.png">

...rather than like this...

<img width="666" alt="image" src="https://user-images.githubusercontent.com/2487968/125120526-88e53f00-e0c0-11eb-827a-505e66258f2a.png">

The recommendation itself is in backticks, so GitHub might not be interpreting my meaning.

Reply:
That makes sense, thank you!